sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -174,11 +174,17 @@ export function useFormioDropdownLoader() {
   };
 
   const loadProgramYear = async (form: any, dropdownName: string) => {
-    return loadDropdown(
-      form,
-      dropdownName,
-      ProgramYearService.shared.getProgramYearOptions(),
+    // Find the dropdown to be populated with the locations.
```

Comment:
This method duplicates the logic from the `loadDropdown`.
Can we adapt the loadDropdown as below?
```ts
const { getFirstComponent } = useFormioUtils();
  const loadDropdown = async (
    form: FormIOForm,
    targetDropdown: string | FormIOComponent,
    loadMethod: Promise<OptionItemAPIOutDTO[]> | OptionItemAPIOutDTO[],
  ): Promise<void> => {
    // Find the dropdown to be populated with the locations.
    let dropdown: FormIOComponent;
    if (typeof targetDropdown === "string") {
      dropdown = getFirstComponent(form, targetDropdown);
    } else {
      dropdown = targetDropdown;
    }
    const optionsItems = await loadMethod;
    dropdown.component.data.values = optionsItems.map(
      (item: OptionItemAPIOutDTO) => ({
        value: item.id,
        label: item.description,
      }),
    );
    dropdown.redraw();
  };
```

Reply:
I have changed it to use `getComponent` to ensure the old behavior as discussed.